libs/eo/transfers/src/lib/eo-transfers.component.ts

```diff
@@ -31,8 +31,15 @@ import { WattCardComponent } from '@energinet-datahub/watt/card';
   template: `
     <eo-popup-message *ngIf="error$ | push"></eo-popup-message>
     <watt-card class="watt-space-stack-l">
-      <h3 class="watt-space-stack-m">This is the beginning</h3>
```

Comment:
Do Søren want to have this information on the Transfer Agreement page?

Reply:
It was there to begin with, i just updated it. Will check up on it.